packages/react/src/utils/interactions/useClick.ts

```diff
@@ -1,7 +1,7 @@
 'use client';
 import * as React from 'react';
-import type { ElementProps, FloatingRootContext } from '@floating-ui/react';
-import { isMouseLikePointerType } from '@floating-ui/react/utils';
+import type { ElementProps, FloatingRootContext } from '../../floating-ui-react';
+import { isMouseLikePointerType } from '../../floating-ui-react/utils';
 import { useAnimationFrame } from '../useAnimationFrame';
 
 const EMPTY_OBJECT = {};
```

Comment:
These `utils/interactions` hooks can go in the floating-ui folder?

Reply:
I plan to move all of those to `utils/interactions` in a later PR.